pkg/util/promqlext/promqlext.go

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package promqlext
+
+import (
+	"github.com/prometheus/prometheus/promql"
+	"github.com/prometheus/prometheus/promql/parser"
+)
+
+// ExtendPromQL enriches PromQL with Mimir extensions.
+func ExtendPromQL() {
+	// Keep an alias for users using holt_winters, even though dropped in Prometheus v3.
+	promql.FunctionCalls["holt_winters"] = promql.FunctionCalls["double_exponential_smoothing"]
```

Comment:
Should we start a deprecation path for this function?

Reply:
I don't know, I don't think we discussed it. Could you remind me of how to do so?

Reply:
Discussed offline that we'll discuss eventual deprecation after this PR.

Reply:
We have a list of deprecated features in `about-versioning.md`. Anyway, not blocking this PR on that.

Reply:
Thanks @colega!